aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -538,6 +540,27 @@ inline void validate_sdpa_input(
   }
   return;
 }
+
+// FlashAttentionV2 requires that head dimension be a multiple of 8
+// This was previously done within the kernel, however
+// This causes the kernel to maybe alias query, key, value
```

Comment:
What does "alias query, key, value" mean? And what is "composite region" in the next comment?